app.py

```diff
@@ -49,7 +56,47 @@ def asynchronous_push(f_stop):
 asynchronous_push(f_stop)
 
 # Now let's run the app!
-chatbot = pipeline(model="microsoft/DialoGPT-medium")
+prompt = load_prompt(PROMPT_TEMPLATES / "openai_chatgpt.json")
```

Comment:
Could also set the template as an env variable so we can toggle without having to hardcode the result